README.md

```diff
@@ -48,6 +48,11 @@ This installs code in two sub-folders. One for the main application (`td`) and o
 
 `npm install`
 
+### Docker
+To run Threat Dragon using docker, configure your environment using dotenv as described in [setup-env.md](setup-env.md) and run the following from the root of the project:
+- `docker build -t owasp-threat-dragon:dev .`
+- `docker run -it -p 3000:3000 $(pwd)/.env:/app/td.server/.env owasp-threat-dragon:dev`
```

Comment:
@lreading does the .env parameter need a "-v" or "--volume list" ? I am running on MacOS and this is required.

I have not tried it on Windows, so this may be different on that platform ... I must dig out my Windows machine soon

Interestingly I get docker error "invalid reference format: repository name must be lowercase" ... a misleading error due to my path has an uppercase character

Reply:
Good catch, my apoligies, bad copy/pasta. This needs to be updated.